recipe/meta.yaml

```
@@ -21,9 +21,14 @@ build:
   skip: True
   {% endif %}
   script:
-    - cp {{ os.environ["FEEDSTOCK_ROOT"] }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt            # [linux]
-    - cp ${RECIPE_DIR}/../LICENSE.txt ${RECIPE_DIR}/LICENSE.txt                              # [osx]
-    - COPY "%RECIPE_DIR%\\..\\LICENSE.txt" "%RECIPE_DIR%\\LICENSE.txt"                       # [win]
+    {% if "FEEDSTOCK_ROOT" in os.environ %}
+    - cp {{ os.environ["FEEDSTOCK_ROOT"] }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt  # [linux]
+    {% else %}
+    - echo '${FEEDSTOCK_ROOT} is undefined. Cannot copy license file'
+    - exit 1
+    {% endif %}
+    - cp ${RECIPE_DIR}/../LICENSE.txt ${RECIPE_DIR}/LICENSE.txt                                  # [osx]
+    - COPY "%RECIPE_DIR%\\..\\LICENSE.txt" "%RECIPE_DIR%\\LICENSE.txt"                           # [win]
```

Forgot we need more Linux selectors

```suggestion
    {% if "FEEDSTOCK_ROOT" in os.environ %}                                                  # [linux]
    - cp {{ os.environ["FEEDSTOCK_ROOT"] }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt            # [linux]
    {% else %}                                                                               # [linux]
    - echo '${FEEDSTOCK_ROOT} is undefined. Cannot copy license file'                        # [linux]
    - exit 1                                                                                 # [linux]
    {% endif %}                                                                              # [linux]
    - cp ${RECIPE_DIR}/../LICENSE.txt ${RECIPE_DIR}/LICENSE.txt                              # [osx]
    - COPY "%RECIPE_DIR%\\..\\LICENSE.txt" "%RECIPE_DIR%\\LICENSE.txt"                       # [win]
```